qlib/workflow/recorder.py

```diff
@@ -422,7 +423,7 @@ def load_object(self, name, unpickler=pickle.Unpickler):
 
         path = None
         try:
-            path = self.client.download_artifacts(self.id, name)
+            path = mlflow.artifacts.download_artifacts(run_id=self.id, artifact_path=name)

```

Comment:
What's the related issue to this PR?
I didn't get the necessity of this change.